lib/modules/platform/gerrit/index.ts

```diff
@@ -397,7 +404,15 @@ export function massageMarkdown(prBody: string): string {
     )
     .replace(
       'you tick the rebase/retry checkbox',
-      'add "rebase!" at the beginning of the commit message.',
+      rebaseLabel
```

Comment:
`rebaseLabel` in Renovate has a default value. I don't think this optionality is necessary here, unless it's your intention that a user _intentionally_ sets it to `null` to disable the capability?

Reply:
Yes, that's what I had in mind. But considering `rebaseLabel` documentation, there's no option to disable it.

![image](https://github.com/user-attachments/assets/2202766c-cea8-4d7e-a0ac-e5130c9a4189)

Thus, I will just remove the optionality, please let me know otherwise.